packages/ens-controller/src/EnsController.ts

```diff
@@ -63,19 +94,35 @@ export class EnsController extends BaseControllerV2<
   EnsControllerState,
   EnsControllerMessenger
 > {
+  ethProvider: Web3Provider | null = null;
```

Comment:
Can this be made private? It doesn't seem as though the extension has any similar public property, nor is there any apparent need for one.

I recall you mentioning in standup that you were intending to make something private, but were blocked by some unit test work. I forget what property that was, maybe this was it?